sdk/anomalydetector/azure-ai-anomalydetector/CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Release History
 
-## 3.0.0b4 (Unreleased)
+## 3.0.0b4 (2021-12-20)
 
-# TODO: Service team fill out
+  **Features**
+  - Introduced the new API: `AnomalyDetectorClientOperationsMixin.last_detect_anomaly`.
+  - Added 2 new optional properties: `impute_mode` & `impute_fixed_value` to `DetectRequest` object.
+  - Added 1 new optional property: `severity` to the `EntireDetectResponse` & `LastDetectResponse` objects.
+  - Removed the optional property `errors` from the `VariableState` object.
+  - Refactored the optional property `contributors` to `interpretation` from the `AnomalyValue` object.
+  - Modified the `FillNAMethod` object into an extensible enum.
```

Comment:
There are not all features. They should be broken out into specific categories as documented here:
https://azure.github.io/azure-sdk/policies_releases.html#change-logs